devenv/bootstrap.py

```diff
@@ -49,7 +49,7 @@ def main(coderoot: str, argv: Sequence[str] | None = None) -> ExitCode:
         # There is a way to perform a headless install but it's more complex
         # (refer to how homebrew does it).
         try:
-            git = proc.run(("xcrun", "-f", "git"), stdout=True)
+            git = proc.run(("xcrun", "-f", "git"))
```

Comment:
if this produces a warning on stderr the result will be incorrect

same for the rest of the calls (this is the reason why they are separate and being separate is a _good_ thing) -- will need to find a different way to get it into the error messages

Reply:
yeah - sorry was working on this late at night / half asleep at the wheel lol